blocks/simple-list-block/features/simple-list/default.jsx

```diff
@@ -142,33 +131,36 @@ const SimpleList = (props) => {
 
 	return (
 		<HeadingSection>
-			<div key={id} className="list-container layout-section">
-				{title ? <Heading className="list-title">{title}</Heading> : null}
+			<Stack key={id} className={BLOCK_CLASS_NAME}>
+				{title ? <Heading className={`${BLOCK_CLASS_NAME}__title`}>{title}</Heading> : null}
 				<Wrapper>
-					{contentElements
-						.reduce(unserializeStory(arcSite), [])
-						.map(({ id: listItemId, itemTitle, imageURL, websiteURL, resizedImageOptions }) => (
-							<React.Fragment key={listItemId}>
-								<StoryItem
-									key={listItemId}
-									id={listItemId}
-									itemTitle={itemTitle}
-									imageURL={imageURL}
-									websiteURL={websiteURL}
-									websiteDomain={websiteDomain}
-									showHeadline={showHeadline}
-									showImage={showImage}
-									resizedImageOptions={resizedImageOptions}
-									placeholderResizedImageOptions={placeholderResizedImageOptions}
-									targetFallbackImage={targetFallbackImage}
-									arcSite={arcSite}
-									imageProps={imageProps}
-								/>
-								<hr />
-							</React.Fragment>
-						))}
+					<Stack className={`${BLOCK_CLASS_NAME}__items`} divider>
+						{contentElements
+							.reduce(unserializeStory(arcSite), [])
+							.map(({ id: listItemId, itemTitle, imageURL, websiteURL, resizedImageOptions }) => (
+								<React.Fragment key={listItemId}>
+									<StoryItem
+										key={listItemId}
+										id={listItemId}
+										classPrefix={BLOCK_CLASS_NAME}
+										itemTitle={itemTitle}
+										imageURL={imageURL}
+										websiteURL={websiteURL}
+										websiteDomain={websiteDomain}
+										showHeadline={showHeadline}
+										showImage={showImage}
+										resizedImageOptions={resizedImageOptions}
+										placeholderResizedImageOptions={placeholderResizedImageOptions}
+										targetFallbackImage={targetFallbackImage}
+										arcSite={arcSite}
+										primaryLogoAlt={primaryLogoAlt}
+									/>
+									{/* <hr /> */}
```

Comment:
nice without the hr! would be able to delete the fragment as well and the comment hr now :)

Reply:
oooops

Reply:
gone now -- nice